solidity/test/LPRewardsTest.js

```diff
@@ -0,0 +1,219 @@
+const {contract, web3, accounts} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
+const {time} = require("@openzeppelin/test-helpers")
+
+const LPRewards = contract.fromArtifact("LPRewards")
+const KeepToken = contract.fromArtifact("KeepToken")
+const WrappedToken = contract.fromArtifact("TestToken")
+
+const BN = web3.utils.BN
+const chai = require("chai")
+chai.use(require("bn-chai")(BN))
+const expect = chai.expect
+
+describe("LPRewards", () => {
+  const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+
+  let keepToken
+  let lpRewards
+  let wrappedToken
+  let rewardDistribution
+  let staker1
+  let staker2
+
+  before(async () => {
+    rewardDistribution = accounts[5]
+    staker1 = accounts[1]
+    staker2 = accounts[2]
```

Comment:
We can declare them as constants earlier - that's how we do it in other tests so it's worth keeping it consistent.

Reply:
done